sql/src/main/java/io/crate/operation/projectors/ShardingShardRequestConsumer.java

```diff
@@ -68,19 +69,20 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-public class ShardingShardRequestAccumulator<TReq extends ShardRequest<TReq, TItem>, TItem extends ShardRequest.Item>
-    implements BatchAccumulator<Row, Iterator<? extends Row>> {
+public class ShardingShardRequestConsumer<TReq extends ShardRequest<TReq, TItem>, TItem extends ShardRequest.Item>
```

Comment:
Not sure if the `Consumer` part makes sense here - it's used as a consumer for a BatchIterator, but it's not consuming ShardRequests - so this may be a bit missleading.